src/constants/externalResources.ts

```diff
@@ -8,6 +8,16 @@ export const wikiUrl = "https://github.com/evergreen-ci/evergreen/wiki";
 
 export const versionControlDocumentationUrl = `${wikiUrl}/Project-and-Distro-Settings#version-control`;
 
+export const patchAliasesDocumentationUrl = `${versionControlDocumentationUrl}#patch-aliases`;
+
+export const pullRequestAliasesDocumentationUrl = `${versionControlDocumentationUrl}#pr-aliases`;
+
+export const commitQueueAliasesDocumentationUrl = `${versionControlDocumentationUrl}#commit-queue-aliases`;
+
+export const gitTagAliasesDocumentationUrl = `${versionControlDocumentationUrl}#git-tag-aliases`;
+
+export const githubChecksAliasesDocumentationUrl = `${versionControlDocumentationUrl}#github-checks-aliases`;
```

Comment:
Looks like these urls don't work I get linked to the following URL when I click the link.
```
https://github.com/evergreen-ci/evergreen/wiki/Project-and-Distro-Settings#version-control%23github-checks-aliases
```
It looks like `versionControlDocumentationUrl` already has an anchor so when you generate these new URLs the second anchor gets auto encoded.